src/ClusterBootstrap/params.py

```diff
@@ -359,7 +359,7 @@
         "CCSAdmins": {
             # The match is in C# Regex Language, please refer to :
             # https://msdn.microsoft.com/en-us/library/az24scfc(v=vs.110).aspx
-            "Allowed": ["jinl@microsoft.com", "hongzl@microsoft.com", "sanjeevm@microsoft.com"],
+            "Allowed": ["jinl@microsoft.com", "hongzl@microsoft.com", "sanjeevm@microsoft.com","zhexu@microsoft.com"],
```

Comment:
change default allowed to "hongzl, anbhu, jachzh, zhexu, dixu,qixcheng"

Reply:
edited

Reply:
Also add "jingzhao,hayua"